firstvoices/backend/tests/test_apis/test_dictionary_api.py

```diff
@@ -62,10 +62,6 @@ def get_valid_data(self, site=None):
             "partOfSpeech": None,
             "pronunciations": [],
             "site": str(site.id),
-            "splitChars": [],
```

Comment:
Removing these fields instead of commenting due to sonar. But leaving the methods to compute these fields. Should be added back with a parameter as a part of [FW-4931](https://firstvoices.atlassian.net/browse/FW-4931)